src/screens/Editor/EditorScreen.export.tsx

```diff
@@ -20,7 +20,7 @@ const {isExport} = window as any;
 const SCROLLBAR_WIDTH = 20;
 
 export const EditorScreen = () => {
-  const [viewMode, setViewMode] = useState<editorModeType>(isExport ? 'doc' : 'editor');
+  const [viewMode] = useState<editorModeType>(isExport ? 'doc' : 'editor');
```

Comment:
Why is `setState` needed here? Because without setter it will always be just a constant.